docs/operate/reference/module-configuration.md

```diff
@@ -227,6 +227,64 @@ For any version type other than **Patch (X.Y.Z)**, the module will upgrade as so
 If, for example, the module provides a motor component, and the motor is running, it will stop while the module upgrades.
 {{% /alert %}}
 
+### Module meta.json configuration
```

Comment:
I really don't think we should add this. I think it will confuse people because this isn't something you put in your machine config, and because I don't want meta.json reference split across many pages.

In my IA PR I've suggested making the meta.json reference into an includes and adding it to a separate page, and if you think this info is key and not redundant, maybe it could live there. But I don't want to conflate meanings of "configure"

Reply:
changed configure to create. We can move it to that file, I don't mind. I think we do just need to now have that somewhere where we can explain about the apps and modules fields